### PR TITLE
feat: set default TTL

### PR DESCRIPTION
**Pull request by makeavish**
30days for metrics, 7 days for traces and logs

### CODE REVIEW

File: exporter/clickhousemetricsexporter/clickhouse.go [hunk 1]
```diff
@@ -100,7 +100,8 @@ func NewClickHouse(params *ClickHouseParams) (base.Storage, error) {
 		)
 		ENGINE = MergeTree
 			PARTITION BY toDate(timestamp_ms / 1000)
-			ORDER BY (metric_name, fingerprint, timestamp_ms);`, database, SAMPLES_TABLE, CLUSTER))
+			ORDER BY (metric_name, fingerprint, timestamp_ms)
+			TTL toDateTime(timestamp_ms) + toIntervalSecond(2592000);`, database, SAMPLES_TABLE, CLUSTER))
```
Comment (srikanthccv) [inline exporter/clickhousemetricsexporter/clickhouse.go:104]: Does this work correcly? Since the timestamp is in epoch millis?

Comment (makeavish) [reply exporter/clickhousemetricsexporter/clickhouse.go:104]: Good catch, let me update it.

Comment (makeavish) [reply exporter/clickhousemetricsexporter/clickhouse.go]: Updated @srikanthccv

**Review by srikanthccv — APPROVED** [review]

**Comment by nityanandagohain** [comment]
Any reason to choose `30days for metrics, 7 days for traces and logs` . Or is it just that we  are expecting more storage to be used by traces and logs?

**Comment by makeavish** [comment]
@nityanandagohain People generally do need to look at metrics older than 7 days for comparison, and it also takes up very less storage than logs and traces. We wanted to set retention low so that updating TTL later won't be costly operation.
We have updated docs and install script to let user know about default retention.

**Pull request merged**